Libraries/oneMKL/matrix_mul/matrix_mul_mkl.cpp

```diff
@@ -0,0 +1,190 @@
+//==============================================================
+// Copyright � 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+#include <CL/sycl.hpp>
+#include <iostream>
+#include <limits>
+#include "mkl.h"
+#include "mkl_blas_sycl.hpp"
+
+using namespace std;
+using namespace cl::sycl;
+
+// Matrix size constants
+#define SIZE 4800  // Must be a multiple of 8.
+#define M SIZE / 8
+#define N SIZE / 4
+#define P SIZE / 2
```

Comment:
`auto constexpr` everywhere

Reply:
Applied:

'auto constexpr SIZE = 4800  // Must be a multiple of 8.
auto constexpr M = SIZE / 8
auto constexpr N = SIZE / 4
auto constexpr P = SIZE / 2'

Is this a correct way or should I do 'auto constexpr double'?

Reply:
If `auto` you cannot precise the type like `double`